scripts/customModels.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python                                                                                                                                  
+import argparse
+import pickle
+import numpy as np
+import mars_troughs as mt
+from mars_troughs.custom_lag_models import ConstantLag,LinearLag,QuadraticLag,CubicLag,PowerLawLag
+from mars_troughs.custom_acc_models import Linear_Insolation, Quadratic_Insolation,Cubic_Insolation,PowerLaw_Insolation
+from mars_troughs.custom_acc_models import Linear_Obliquity, Quadratic_Obliquity,Cubic_Obliquity,PowerLaw_Obliquity
+from mars_troughs.datapaths import load_insolation_data, load_obliquity_data
+import cProfile
+import sys
+
+def main():
+        
+    p=argparse.ArgumentParser(description='Parse submodel numbers for MCMC')
```

Comment:
Don't include it in this PR, but I recommend switching to [configparser](https://docs.python.org/3/library/configparser.html) instead of using `argparse`. This way you can put your information into a "config file" (a formatted text file, essentially). IMO it can be annoying to type out long things on the command line, and a config parser mitigates that.